src/client/src/services/model/trade.js

```diff
@@ -35,7 +36,8 @@ export default class Trade {
     this._tradeDate = tradeDate;
     this._valueDate = valueDate;
     this._status = status;
-    this._isNew = isNew;
+    this._isNew = !isStateOfTheWorld;
```

Comment:
No, isNew is used to flash the newly added row in the blotter. It's used for a different purpose than the isStateOfTheWorld, but the initial value is set by the value of the 'isStateOfTheWorld'